.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+- "2.7"
```

Comment:
But since we don't have any intention to add support for Py 2.7, I don't see why we should keep it there.